yb-voyager/src/srcdb/oracle.go

```diff
@@ -174,6 +174,60 @@ func (ora *Oracle) ExportSchema(exportDir string) {
 	ora2pgExtractSchema(ora.source, exportDir)
 }
 
+// return list of jsons having index info like index name, index type, table name, column name
+func (ora *Oracle) GetIndexesInfo() *[]utils.IndexInfo {
+	// TODO(future): once we implement table-list/object-type for export schema
+	// we will have to filter out indexes based on tables or object types that are not being exported
```

Comment:
I think there is a PR up for this already? Pls do not miss adding support for this, either as part of this PR itself, or a subsequent one!

Reply:
need to wait for that PR to merge.
Created a high-priority ticket: https://yugabyte.atlassian.net/browse/DB-8261